### PR TITLE
Add GAE Flex websockets sample

### PR DESCRIPTION
**Pull request by ace-n**
@grayside @andrewsg FYI

### CODE REVIEW

**Comment by ace-n** [comment]
**Note** (mostly to myself): some of the packages used here aren't compatible with older versions of PHP (and are causing CI failures).

File: appengine/flexible/websockets/composer.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+    "require": {
+        "cboden/ratchet": "^0.4.1",
+        "guzzlehttp/psr7": "^1.5",
+        "silex/silex": "^2.3",
```
Comment (grayside) [inline appengine/flexible/websockets/composer.json]: Silex is end-of-life and should not be used.
Newer samples might use raw PHP or Slim. @bshaffer has also been looking at Lumen for bookshelf.

File: appengine/flexible/websockets/index.html.twig [hunk 2]
```diff
@@ -0,0 +1,88 @@
+{#
```
Comment (grayside) [inline appengine/flexible/websockets/index.html.twig]: This file doesn't seem to be using Twig? Maybe just use pure HTML and drop the Twig dependency.

File: appengine/flexible/websockets/index.html.twig [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+# Copyright 2019 Google LLC
```
Comment (grayside) [inline appengine/flexible/websockets/index.html.twig]: I overhauled this HTML for the Go sample (fusing the ideas in Python and Nodejs and adding some additional styling), feel free to copy [golang-samples index.html](https://github.com/GoogleCloudPlatform/golang-samples/blob/master/appengine_flexible/websockets/static/index.html), screenshot in [golang-samples PR #837](https://github.com/GoogleCloudPlatform/golang-samples/pull/837)

File: appengine/flexible/websockets/composer.json (hunk 1 above)
Comment (ace-n) [reply appengine/flexible/websockets/composer.json]: Chatted offline - @bshaffer is fine with this, so I'm marking as resolved for now.

File: appengine/flexible/websockets/index.html.twig (hunk 3 above)
Comment (ace-n) [reply appengine/flexible/websockets/index.html.twig]: Done.

File: appengine/flexible/websockets/index.html.twig (hunk 2 above)
Comment (ace-n) [reply appengine/flexible/websockets/index.html.twig]: Done.

File: appengine/flexible/websockets/socket-server.php [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+$port = 8000;
+
+$server = IoServer::factory(
+    new HttpServer(
+        new WsServer(
+                    $socketDemo
+                )
+            ),
+    $port
```
Comment (grayside) [inline appengine/flexible/websockets/socket-server.php]: trivial bit of whitespace here.

Comment (ace-n) [reply appengine/flexible/websockets/socket-server.php]: Fixed - surprised that `php-cs-fixer` didn't handle this.

**Comment by ace-n** [comment]
> **Note** (mostly to myself): some of the packages used here aren't compatible with older versions of PHP (and are causing CI failures).

This is now a **do not merge** until @bshaffer signs off.

**Pull request merged**